yb-voyager/cmd/import.go

```diff
@@ -278,7 +278,7 @@ func validateTargetPortRange() {
 }
 
 func validateTargetSchemaFlag() {
-	// do not check for source and source-replica imports
+	// we want the check to run for import data to target and import schema commands
```

Comment:
@atharva-kurhade-yb let's add some more detail just so that we all don't forget this in future:  
`we want to run this check only for import-data-to-target and import-schema commands. This is not applicable for import-data-to-source-replica (validateFFDBSchemaFlag) /import-data-to-source (no ability to pass schema). For import-data-file,  we allow this flag and source is PG(dummy)`